components/bannerbear/actions/create-image/create-image.mjs

```diff
@@ -0,0 +1,40 @@
+import { axios } from "@pipedream/platform";
+
+import bannerbear from '../../bannerbear.app.mjs'
+
+export default {
+  key: "bannerbear-create-image",
+  name: "Create an Image",
+  description: "Create an image using template and modifications. [See the docs](https://developers.bannerbear.com/#post-v2-images)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    bannerbear,
+    template: {
+      propDefinition: [
+        bannerbear,
+        "template",
+      ],
+    },
+    modifications: {
+      type: 'object',
+      description: 'A list of modifications you want to make. [See the docs](https://developers.bannerbear.com/#post-v2-images)'
+    }
+  },
+  async run({ $ }) {
+    const response = await axios($, {
```

Comment:
Could you please move your call to the `app`? Normally we use the `app` as a repository for all API calls. Take a look at `blogger` components for a reference